src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/diff/PrisonerDifferenceService.kt

```diff
@@ -93,7 +97,7 @@ class PrisonerDifferenceService(
       previousPrisonerSnapshot?.also {
         val differences = getDifferencesByCategory(it, prisoner)
         if (differences.isEmpty()) {
-          raiseNoDifferencesTelemetry(offenderBooking.offenderNo)
+          raiseNoDifferencesFoundTelemetry(offenderBooking.offenderNo)
         } else {
```

Comment:
raise a different event when there should be differences when there isn't compared to there is no differences at all